samples/display/ili9340/README.rst

```diff
@@ -0,0 +1,18 @@
+.. _ili9340-sample:
+
+ILI9340 Display driver
+######################
+
+Overview
+********
+
+Sample application that alternates LCD corners between RGB.
```

Comment:
I don't know what this means. (The corner pixels of the LCD will cycle from Red to Green to Blue every half-second?)  Please explain what the sample is showing on the display. Also, we have a template for samples (doc/samples/sample.tmpl) that you should follow to explain more about how to setup and build your sample and  mention this sample is for a specific board.